pkg/networkmanager/interfaces.js

```diff
@@ -41,7 +41,7 @@ import 'bootstrap/dist/js/bootstrap';
 import "page.scss";
 import "table.css";
 import "plot.css";
-import "./networking.css";
+import "./networking.scss";
```

Comment:
I needed to rename the file. The contents were SCSS but the filename was CSS still.